### DISCUSSION TITLE
Health scanner useless for user itself. Unrealistic.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#12479] [Ideas and suggestions]
Health scanner does not show infection and poisoning on whoever uses it. You cannot scan yourself. It's unrealistic.
It seems to me that with an advanced diagnostic tool, anyone can scan themselves.

**Comment by mygamingaccount** [comment]
Nobody ever cared about this.

Quick! Put this on and tell me if I have para*mmmphpmhpmhpmh....*
